sormas-base/setup/s2s-import-to-truststore.sh

```diff
@@ -127,4 +142,26 @@ if [[ ${NEW_TRUSTSTORE} = true ]]; then
   } >> "${SORMAS_PROPERTIES}"
 fi
 
+echo "Updating server list CSV"
+
+( head -1 "$CSV_FILE" ) >> "${ORGANIZATION_LIST_FILE}"
+
+echo "Importing SSL certificate"
```

Comment:
This should better not be done in the script because it is only relevant for test systems.